### ISSUE TITLE
[ESLINT] can't resolve reference #/definitions/directiveConfigSchema from id #

### ISSUE THREAD

**Issue by zeronerorgb** [sveltejs/kit#6168]
### Describe the bug

Hello
Eslint don't work on demo app

A few weeks ago it's works fine

### Reproduction

I create demo app like shown below
```
npm create svelte test

create-svelte version 2.0.0-next.159

Welcome to SvelteKit!

This is beta software; expect bugs and missing features.

Problems? Open an issue on https://github.com/sveltejs/kit/issues if none exists already.

✔ Which Svelte app template? › SvelteKit demo app
✔ Add type checking with TypeScript? › Yes, using TypeScript syntax
✔ Add ESLint for code linting? … No / Yes
✔ Add Prettier for code formatting? … No / Yes
✔ Add Playwright for browser testing? … No / Yes

Your project is ready!
✔ Typescript
  Inside Svelte components, use <script lang="ts">
✔ ESLint
  https://github.com/sveltejs/eslint-plugin-svelte3
✔ Prettier
  https://prettier.io/docs/en/options.html
  https://github.com/sveltejs/prettier-plugin-svelte#options
✔ Playwright
  https://playwright.dev

Install community-maintained integrations:
  https://github.com/svelte-add/svelte-adders

Next steps:
  1: cd test
  2: npm install (or pnpm install, etc)
  3: git init && git add -A && git commit -m "Initial commit" (optional)
  4: npm run dev -- --open

To close the dev server, hit Ctrl-C

Stuck? Visit us at https://svelte.dev/chat
```
install all dependencies and run  eslint `yarn install && yarn lint` 

after that i see
```
yarn install && yarn lint
yarn install v1.22.19
warning package.json: "test" is also the name of a node core module
warning package.json: No license field
info No lockfile found.
warning test@0.0.1: "test" is also the name of a node core module
warning test@0.0.1: No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
Done in 13.73s.
yarn run v1.22.19
warning package.json: "test" is also the name of a node core module
warning package.json: No license field
$ prettier --check . && eslint .
Checking formatting...
All matched files use Prettier code style!
can't resolve reference #/definitions/directiveConfigSchema from id #
can't resolve reference #/definitions/directiveConfigSchema from id #
can't resolve reference #/definitions/directiveConfigSchema from id #
can't resolve reference #/definitions/directiveConfigSchema from id #
Done in 8.21s.
```

A few weeks ago it's works fine

### System Info

```shell
System:
    OS: Linux 5.4 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (8) x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
    Memory: 23.88 GB / 24.98 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 18.0.0 - ~/.nvm/versions/node/v18.0.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.0.0/bin/yarn
    npm: 8.6.0 - ~/.nvm/versions/node/v18.0.0/bin/npm
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.65
    @sveltejs/kit: next => 1.0.0-next.428
    svelte: ^3.46.0 => 3.49.0
    vite: ^3.0.8 => 3.0.9
```

### Severity

annoyance

**Comment by tcc-sejohnson** [comment]
Confirmed that I'm seeing the same "can't resolve reference" errors, though eslint doesn't seem to be actually failing, just printing them as warnings.

**Comment by Valexr** [comment]
Same problem in eslint watch mode:

```
www dev$ vite dev
packages/animation dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/animation dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/animation dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/animation dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/easing dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/easing dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/easing dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/easing dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/core dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/core dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/core dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/core dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/media dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/media dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/media dev: can't resolve reference #/definitions/directiveConfigSchema from id #
packages/media dev: can't resolve reference #/definitions/directiveConfigSchema from id #
```

**Comment by dhartoto** [comment]
Hi all, not sure if this is helpful for you but I got this page by searching for resolution for the same error when bumping `@typescript-eslint/eslint-plugin@5.33.0` to @typescript-eslint/eslint-plugin@5.34.0`.

Good luck!

**Comment by dummdidumm** [comment]
Sounds like an upstream bug then

**Comment by dummdidumm** [comment]
This should be fixed in `@typescript-eslint/eslint-plugin@5.35.1`
